reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -122,7 +120,7 @@ class FreemarkerTemplateProcessor(
             "ortResult" to input.ortResult,
             "licenseTextProvider" to input.licenseTextProvider,
             "LicenseView" to LicenseView,
-            "helper" to TemplateHelper(input.ortResult, input.licenseClassifications, input.resolutionProvider),
+            "helper" to TemplateHelper(input),
```

Comment:
This change changes the default value from `hint` to `warning`. Is it intended?

Reply:
Yes. That's why I marked this as a breaking change. But AFAIK that's what HERE wants (i.e. you don't require hints to be addressed), and it's also what we decided to settle on now, too.

Reply:
How about mentioning that default value change in the commit message?

Reply:
Makes sense, done.

Reply:
> But AFAIK that's what HERE wants (i.e. you don't require hints to be addressed), and it's also what we decided to settle on now, too.

Almost, we also don't require warnings to be addressed, but that's ok as it is configurable.

Reply:
> Almost, we also don't require warnings to be addressed, but that's ok as it is configurable.

Do you need to make a configuration change then? If so, please tell me (or only approve by then) when you're good to merge this.

Reply:
You can merge it now. Thanks for asking!